frontend/public/components/utils/resource-log.tsx

```diff
@@ -286,118 +412,105 @@ export const LogControls: React.FC<LogControlsProps> = ({
-              toggleWrapLines(checked);
-            }}
-          />
-          <Divider
-            orientation={{
-              default: 'vertical',
-            }}
-          />
-          <a href={currentLogURL} target="_blank" rel="noopener noreferrer">
-            <OutlinedWindowRestoreIcon className="co-icon-space-r" />
-            {t('public~Raw')}
-          </a>
-          <Divider
-            orientation={{
-              default: 'vertical',
+        {isMobile ? (
+          <Dropdown
+            isOpen={isDropdownOpen}
+            onSelect={onDropdownSelect}
+            onOpenChange={(isOpen: boolean) => setIsDropdownOpen(isOpen)}
+            toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+              <MenuToggle
+                ref={toggleRef}
+                onClick={onDropdownToggleClick}
+                isExpanded={isDropdownOpen}
+                variant="plain"
+                className="pf-v5-u-mt-xs"
+                aria-label={t('public~Dropdown toggle')}
+              >
+                <EllipsisVIcon />
+              </MenuToggle>
+            )}
+            shouldFocusToggleOnSelect
+            popperProps={{
+              position: 'right',
             }}
-          />
-          <a href={currentLogURL} download={`${resource.metadata.name}-${containerName}.log`}>
-            <DownloadIcon className="co-icon-space-r" />
-            {t('public~Download')}
-          </a>
-          {screenfull.enabled && (
-            <>
-              <Divider
-                orientation={{
-                  default: 'vertical',
-                }}
-              />
-              <Button variant="link" isInline onClick={toggleFullscreen}>
-                {isFullscreen ? (
-                  <>
-                    <CompressIcon className="co-icon-space-r" />
-                    {t('public~Collapse')}
-                  </>
-                ) : (
-                  <>
-                    <ExpandIcon className="co-icon-space-r" />
-                    {t('public~Expand')}
-                  </>
+          >
+            <DropdownGroup label={t('public~Log actions')}>
+              <DropdownList>
+                {!_.isEmpty(podLogLinks) && renderPodLogLinks()}
+                <DropdownItem
+                  onClick={(e) => {
+                    e.preventDefault();
+                    toggleShowFullLog(!isShowFullLog);
+                  }}
+                >
+                  {fullLog}
+                </DropdownItem>
+                <DropdownItem
+                  onClick={(e) => {
+                    e.preventDefault();
+                    toggleWrapLines(!isWrapLines);
+                  }}
+                >
+                  {wrapLines}
+                </DropdownItem>
+                <DropdownItem to={currentLogURL} isExternalLink>
+                  {raw}
+                </DropdownItem>
+                <DropdownItem to={currentLogURL} isExternalLink>
```

Comment:
```suggestion
                <DropdownItem to={currentLogURL} download={`${resource.metadata.name}-${containerName}.log`>
```
Plus a forthcoming version of `@patternfly/react-core` that includes https://github.com/patternfly/patternfly-react/pull/9892